applications/nrf5340_audio/src/modules/dfu_entry.c

```diff
@@ -111,9 +99,14 @@ void dfu_entry_check(ble_init_func ble_init)
 
 	if (pressed) {
 		LOG_INF("Enter SMP_SVR service only status");
-		ret = ble_init(on_ble_core_ready_dfu_entry);
+		ret = ble_init(false);
```

Comment:
It likely makes most sense not to have the watchdog enabled when performing DFU.

Reply:
Decided to wait until dfu_entry.c has been moved to bluetooth folder